espflash/src/image_format/esp_idf.rs

```diff
@@ -213,8 +213,10 @@ impl<'a> IdfBootloaderFormat<'a> {
 
         let mut data = bytes_of(&header).to_vec();
 
-        let flash_segments: Vec<_> = merge_adjacent_segments(rom_segments(chip, &elf).collect());
-        let mut ram_segments: Vec<_> = merge_adjacent_segments(ram_segments(chip, &elf).collect());
+        let flash_segments: Vec<_> =
```

Comment:
I think a comment here explaining why we pad align _after_ would be good so we don't revert this inadvertently at some point